Configuration/Services.yaml

```
@@ -66,6 +66,13 @@ services:
         identifier: 'cart-products--check-product-availability'
         event: Extcode\Cart\Event\CheckProductAvailabilityEvent
 
+  Extcode\CartProducts\EventListener\Order\Stock\ProcessOrderCheckStock:
+    tags:
+      - name: event.listener
+        identifier: 'cart-products--process-order-check-stock'
```

For identifier I reflect the name and the namespace of the class (see: `cart-products--order--stock--handle-stock`).

```suggestion
        identifier: 'cart-products--order--stock--check-stock'
```